nuxeo-services/nuxeo-permissions/src/main/resources/OSGI-INF/listeners-contrib.xml

```diff
@@ -8,6 +8,11 @@
       <event>documentSecurityUpdated</event>
     </listener>
 
+    <listener name="permissionGarbageCollectorListener" async="true" postCommit="false"
```

Comment:
what about?
```enabled="${nuxeo.aceinfo.gc.enabled:=false}"```
this will save the creation and the submit of a ListenerWork in the common queue, that will be run to do nothing, also, it will work using nuxeo.conf which is not the case with your contrib.